library/src/main/java/com/meisolsson/githubsdk/model/GitHubCommentBase.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright 2015 Henrik Olsson
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.meisolsson.githubsdk.model;
+
+import android.os.Parcelable;
+import android.support.annotation.Nullable;
+
+import com.meisolsson.githubsdk.core.FormattedHtml;
+import com.meisolsson.githubsdk.core.FormattedTime;
+import com.squareup.moshi.Json;
+
+import java.util.Date;
+
+public abstract class GitHubCommentBase implements Parcelable {
```

Comment:
We should probably comment on why this is its own file. Since everything from the looks of it could extend from `GitHubComment`. From what I remember we can't extend from an `@AutoValue` annotated class?

Reply:
Exactly that is the reason why I put it in a separate class.